rules/prefer-string-raw.js

```diff
@@ -64,6 +61,45 @@ const create = context => {
 			},
 		};
 	});
+
+	context.on('TemplateLiteral', node => {
+		if (node.parent.type === 'TaggedTemplateExpression') {
+			return;
+		}
+
+		let hasBackslash = false;
+
+		for (const quasi of node.quasis) {
+			const {raw, cooked} = quasi.value;
+
+			if (cooked.at(-1) === BACKSLASH) {
+				return;
+			}
+
+			const unescapedQuasi = unescapeBackslash(raw);
+			if (unescapedQuasi !== cooked) {
+				return;
+			}
+
+			if (cooked.includes(BACKSLASH)) {
+				hasBackslash = true;
+			}
+		}
+
+		if (!hasBackslash) {
+			return;
+		}
+
+		return {
+			node,
+			messageId: MESSAGE_ID,
+			* fix(fixer) {
+				yield * fixSpaceAroundKeyword(fixer, node, context.sourceCode);
+				yield * node.quasis.map(quasi => replaceTemplateElement(fixer, quasi, quasi.value.cooked));
```

Comment:
Let's use a for loop to avoid creating a new array, we should also skip unnecessary replacements.